tests/app/services/test_process_request_json.py

```diff
@@ -162,21 +168,22 @@ def test_missing_field_in_transformation():
                 ]
             }
         },
-    ], TEXT_FIELDS_SET=set(['someField', 'otherField'])):
-        request = {
-            "otherField": ["wibble"],
-        }
+    ]
+    services_mapping.text_fields_set = set(['someField', 'otherField'])
+    request = {
+        "otherField": ["wibble"],
+    }
 
-        result = convert_request_json_into_index_json(request)
-        # really just checking this case doesn't throw!
+    result = convert_request_json_into_index_json(request)
+    # really just checking this case doesn't throw!
 
-        assert result == {
-            "otherField": ["wibble"],
-        }
+    assert result == {
```

Comment:
Man, yes. You're right. Now my 20 second fix led to us both wasting 10 minutes. I need to learn to keep quiet sometimes. 🙊